### PR TITLE
TFLM: To support Himax WE1 EVB

### PR DESCRIPTION
**Pull request by CKHSUinHimax**
In this pull request support Himax WiseEye Plus platform is added.

### CODE REVIEW

**Comment by CKHSUinHimax** [comment]
@googlebot I signed it!

**Comment by CKHSUinHimax** [comment]
@googlebot I fixed it.

**Comment by bigcat-himax** [comment]
@googlebot I consent.

**Review by bigcat-himax — COMMENTED** [review]
The modification is okay. In this pull request support Himax WiseEye Plus platform is added.

**Comment by bigcat-himax** [comment]
@googlebot I consent.

File: tensorflow/lite/micro/examples/person_detection_experimental/himax_we1_evb/image_provider.cc [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+hx_drv_sensor_image_config_t g_pimg_config;
+
+
+TfLiteStatus GetImage(tflite::ErrorReporter* error_reporter, int image_width,
+                      int image_height, int channels, int8_t* image_data) {
+  static bool is_initialized = false;
+
+  if (!is_initialized) {
+    if(hx_drv_sensor_initial(&g_pimg_config)!= HX_DRV_LIB_PASS)
+    {
```
Comment (petewarden) [inline tensorflow/lite/micro/examples/person_detection_experimental/himax_we1_evb/image_provider.cc:31]: Can you run something like clang-format with the google style option over this code, to make sure it passes our internal formatting tests?

File: tensorflow/lite/micro/examples/person_detection_experimental/himax_we1_evb/main_functions.cc [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+// In order to use optimized tensorflow lite kernels, a signed int8 quantized
+// model is preferred over the legacy unsigned model format. This means that
+// throughout this project, input images must be converted from unisgned to
+// signed format. The easiest and quickest way to convert from unsigned to
+// signed 8-bit integers is to subtract 128 from the unsigned value to get a
+// signed value.
+
+// An area of memory to use for input, output, and intermediate arrays.
+constexpr int kTensorArenaSize = 125 * 1024;
+#pragma Bss(".tensor_arena")
```
Comment (petewarden) [inline tensorflow/lite/micro/examples/person_detection_experimental/himax_we1_evb/main_functions.cc:45]: As discussed on the call, can you confirm what the differences are between this and the regular main_functions.cc file? If the differences are minor, let's figure out what the simplest way to include them without a full copy is.

**Review by petewarden — CHANGES_REQUESTED** [review]

**Review by petewarden — APPROVED** [review]

**Pull request merged**